DopplerCurrencyJob/DopplerCurrencyService/DopplerCurrencyService.cs

```diff
@@ -59,41 +59,39 @@ public async Task<IList<CurrencyResponse>> GetCurrencyByCode()
                 try
                 {
                     var cstTime = TimeZoneInfo.ConvertTimeFromUtc(DateTime.UtcNow, cstZone);
-                    var retryCount = 0;
-
                     var httpResponse = await GetCurrencyValue(currencyCode, cstTime);
                     var jsonResult = await httpResponse.Content.ReadAsStringAsync();
-                    var isHoliday = jsonResult.Contains("No USD for this date") || jsonResult.Contains("Html Error Mxn currency");
 
-                    if (!isHoliday && !httpResponse.IsSuccessStatusCode )
-                    {
-                        _logger.LogError("{ReasonPhrase}. Error getting currency for {currencyCode}.", httpResponse.ReasonPhrase, currencyCode);
-                        continue;
-                    }
-                    else if (isHoliday)
+                    if (httpResponse.IsSuccessStatusCode)
                     {
-                        while (retryCount < _dopplerCurrencySettings.HolidayRetryCountLimit)
+                        var result = JsonConvert.DeserializeObject<CurrencyResponse>(jsonResult);
+
+                        if (result.CotizationAvailable)
+                        {
+                            returnList.Add(result);
+                        }
+                        else
                         {
-                            cstTime = DateTime.UtcNow.DayOfWeek == DayOfWeek.Monday ? cstTime.AddDays(-3) : cstTime.AddDays(-1);
-                            httpResponse = await GetCurrencyValue(currencyCode, cstTime);
-                            if (httpResponse.IsSuccessStatusCode)
+                            _logger.LogError("{ReasonPhrase}. Error getting currency for {currencyCode}.", httpResponse.ReasonPhrase, currencyCode);
+
+                            ///get the most recent previous price
```

Comment:
```suggestion
                            // Get the most recent previous price
```

Reply:
Ok, I will change it

Reply:
I think that this change was not applied, anyway is not important